### ISSUE TITLE
Error during sylius:install

### ISSUE THREAD

**Issue by gmichard** [Sylius/Sylius#14198]
**Sylius version affected**: 1.11.7

**Description**  
Error during `php bin/console sylius:install`.  
More precisely, the error occurs during the `sylius:install:database` command.

**My configuration**  
- OS : MacOS
- PHP version : `PHP 8.1.8`
- DBMS : MariaDB `mariadb  Ver 15.1 Distrib 10.8.3-MariaDB, for osx10.15 (x86_64)`

**Error output**  
```
Step 1 of 5. Checking system requirements.
------------------------------------------

Success! Your system can run Sylius properly.

Step 2 of 5. Setting up the database.
-------------------------------------

Creating Sylius database for environment dev.
It appears that your database already exists.
Warning! This action will erase your database.
Would you like to reset it? (y/N) y
 1/4 [░░░░░░░░                    ]  25%
In AbstractMySQLDriver.php line 128:
                                                                                                                                                                                     
  An exception occurred while executing 'ALTER TABLE sylius_adjustment ADD CONSTRAINT FK_ACA6E0F28D9F6D38 FOREIGN KEY (order_id) REFERENCES sylius_order (id) ON DELETE CASCADE':    
                                                                                                                                                                                     
  SQLSTATE[HY000]: General error: 1025 Error on rename of './sylius_dev/sylius_adjustment' to './sylius_dev/#sql-backup-273-9cc' (errno: 168 "Unknown (generic) error from engine")  
                                                                                                                                                                                     

In Exception.php line 18:
                                                                                                                                                                                     
  SQLSTATE[HY000]: General error: 1025 Error on rename of './sylius_dev/sylius_adjustment' to './sylius_dev/#sql-backup-273-9cc' (errno: 168 "Unknown (generic) error from engine")  
                                                                                                                                                                                     

In PDOConnection.php line 141:
                                                                                                                                                                                     
  SQLSTATE[HY000]: General error: 1025 Error on rename of './sylius_dev/sylius_adjustment' to './sylius_dev/#sql-backup-273-9cc' (errno: 168 "Unknown (generic) error from engine")  
                                                                                                                                                                                     

doctrine:migrations:migrate [--write-sql [WRITE-SQL]] [--dry-run] [--query-time] [--allow-no-migration] [--all-or-nothing [ALL-OR-NOTHING]] [--configuration CONFIGURATION] [--em EM] [--conn CONN] [--] [<version>]
```

**Steps to reproduce**  
```sh
composer create-project sylius/sylius-standard acme
php bin/console sylius:install
```

---

Is it related to MariaDB? Instead of MySQL?
Thanks for your advises. 👍

**Comment by Romaixn** [comment]
Hello @gmichard ! It seems that the database was not empty, did you check if it did not already exist? (With data from another installation for example)

**Comment by CoderMaggie** [comment]
@gmichard 👋🏻 Please try finding help with installation on our [Slack channel](http://sylius.org/slack). If the problem is a bug, please reopen this issue.

**Comment by gmichard** [comment]
> Hello @gmichard ! It seems that the database was not empty, did you check if it did not already exist? (With data from another installation for example)

@Romaixn  Not the cause here. :/
In fact, Sylius is able to create the database or erase the existing one if necessary (after confirmation). I tried both.

**Comment by gmichard** [comment]
More traces about the error that occurs during the first "ALTER TABLE" of the first migration script ("Version20161202011555").

- [logs-sylius-install.txt](https://github.com/Sylius/Sylius/files/9866684/install.txt)
- [logs-doctrine-migrate.txt](https://github.com/Sylius/Sylius/files/9866682/doctrine-migrate.txt)

It's probably more related to my environment than Sylius.
Keep this issue closed.
Thanks for the help.

**Comment by Javiervillag99** [comment]
Did you solve this? I am struggling with the same error.

**Comment by gmichard** [comment]
@Javiervillag99 Yes, I solved this.

As supposed, it wasn't directly related to Sylius.
It's been a while but **I had to properly reinstall MariaDB** (by using `brew `).
I probably had also reinstall PHP 8, for consistency (but I don't think that this solved my problem).
Then, I did a fresh install of Sylius (new download of packages, in a new directory, with a new database).
And it worked.

Nevertheless, before the clean reinstall of MariaDB (and PHP), I had the same (latest) version of MariaDB installed (kept up to date). But, I suppose that the was a trouble during an update.

I have no more explainations. :/
Hope that helps.
